packages/cli/src/commands/ui/create/vue.ts

```diff
@@ -80,9 +80,9 @@ export default class Vue extends Command {
 
   private async invokePlugin(applicationName: string) {
     const cfg = await this.configuration.get();
-    const {providerUsername} = await this.getUserInfo();
+    const {userInfo, apiKey} = await this.platformUserCredentials();
```

Comment:
`getUserInfo()` is now orphaned, should be deleted

Reply:
oups. forgot to delete it. thanks